VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitFragment.java

```diff
@@ -14,6 +14,8 @@
 public abstract class CourseUnitFragment extends BaseFragment implements PageViewStateCallback, RunnableCourseComponent {
     public interface HasComponent {
         CourseComponent getComponent();
+        boolean navigateNextComponent();
```

Comment:
If nothing uses it now and there is no concrete plan or reason to use it in the future, than remove it.